elements/pfe-sass/mixins/_components.scss

```diff
@@ -186,7 +186,7 @@
 @mixin pfe-chevron($state: open, $position: after, $size: .4em) {
   content: "";
   position: absolute;
-  top: calc((#{pfe-var(container-spacer)} * .5) + #{$size});
+  top: calc(#{pfe-var(container-spacer)} + #{$size});
```

Comment:
By using the full container spacer height, doesn't that align the caret too low?

Reply:
It actually looked too high before. This looks much more vertically aligned with the text.

Reply:
Here's the placement on my sandbox:
![Screen Shot 2019-12-03 at 4 27 26 PM](https://user-images.githubusercontent.com/32456107/70091426-11992080-15ea-11ea-9c95-62d343b9c324.png)